include/private/gcconfig.h

```diff
@@ -106,10 +106,20 @@ EXTERN_C_BEGIN
 
 /* And one for QNX: */
 # if defined(__QNX__)
-#    define I386
+#    define QNX_STACKBOTTOM 1
+#    if defined(__arm__) || defined(__ARM__)
```

Comment:
Just making sure that 64 bit arm doesn't have either of these defines? Should we check for 64-bit first as on x86?

Reply:
Good point, let me verify that.

Reply:
Checked it and only one should be defined, but changed it nonetheless because checking first for ARM64 is the safer approach.